tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_creator.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 The TensorFlow Recommenders-Addons Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lint-as: python3
+
+from abc import ABCMeta
+from tensorflow_recommenders_addons import dynamic_embedding as de
+
+
+class KVCreator(object, metaclass=ABCMeta):
+
+    def __init__(self, config=None):
+        self.config = config
+
+    def create(self,
+               key_dtype=None,
+               value_dtype=None,
+               default_value=None,
+               name=None,
+               checkpoint=None,
+               init_size=None,
+               config=None):
+
+        raise NotImplementedError('create function must be implemented')
+
+
+class CuckooHashTableConfig(object):
+
+    def __init__(self):
+        """ CuckooHashTableConfig include nothing for parameter default satisfied.
+    """
+        pass
+
+
+class CuckooHashTableCreator(KVCreator):
+
+    def create(
+        self,
+        key_dtype=None,
+        value_dtype=None,
+        default_value=None,
+        name=None,
+        checkpoint=None,
+        init_size=None,
+        config=None,
+    ):
+        return de.CuckooHashTable(
+            key_dtype=key_dtype,
+            value_dtype=value_dtype,
+            default_value=default_value,
+            name=name,
+            checkpoint=checkpoint,
+            init_size=init_size,
+            config=config,
+        )
+
+
+class RedisTableConfig(object):
+
+    def __init__(
```

Comment:
Is it better to put the redis configuration in an environment variable or configuration file?